static/app/views/starfish/modules/databaseModule/queries.ts

```diff
@@ -322,6 +322,44 @@ export const useQueryPanelTable = (
   });
 };
 
+export const useGetExampleTransaction = (
```

Comment:
Small nit, we prefix everything here with `useQuery`
```suggestion
export const useQueryGetExampleTransaction = (
```